raft/src/main/java/org/apache/kafka/raft/FollowerState.java

```diff
@@ -137,11 +137,11 @@ public boolean updateHighWatermark(OptionalLong highWatermark) {
         return fetchingSnapshot;
     }
 
-    public void setFetchingSnapshot(Optional<RawSnapshotWriter> fetchingSnapshot) {
+    public void setFetchingSnapshot(Optional<RawSnapshotWriter> newSnapshot) {
         if (fetchingSnapshot.isPresent()) {
             fetchingSnapshot.get().close();
         }
```

Comment:
Ok, so previously we were closing whatever snapshot was passed in to `setFetchingSnapshot?`